specification/http/src/main/scripts/org/kaazing/specification/http/rfc7231/redirection/relative.location/request.rpt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+connect http://localhost:8000/resource
+connected
+
+write method "GET"
```

Comment:
I think k3po is sends "GET http://localhost:8000/resource HTTP/1.1" on the wire (instead of "GET /resource HTTP/1.1"). This is not common. But adding "write Host" will generate "GET /resource HTTP/1.1"